### ISSUE TITLE
[Feature Request] Pure Black/Amoled Theme in Dark Mode for OLED

### ISSUE THREAD

**Issue by SagXD** [daniebeler/pixelix#123]
Pure Black theme will be great for OLED device to save battery. 
Thanks for making this app :).

**Comment by piratenpanda** [comment]
Actually pure black does not save much energy compared to dark grey but offers worse accessibility so I'd opt for not ditching the currently available color scheme and to provide am additional option for people who want lure black.

**Comment by SagXD** [comment]
You can add it alongside current Theme. But, if you don't want to I will not force. Thanks

**Comment by piratenpanda** [comment]
I am not the author, just adding comments :) I am all for having more options

**Comment by Hiebeler** [comment]
The new update with the AMOLED Theme is now published, it is already on github releases and the google play store and will be up on F-Droid in the coming days.
